opensrp-pnc/src/main/java/org/smartregister/pnc/repository/PncRegistrationDetailsRepository.java

```diff
@@ -14,10 +14,12 @@
 
 public class PncRegistrationDetailsRepository extends PncDetailsRepository {
 
+    private static final String TABLE = PncDbConstants.Table.PNC_REGISTRATION_DETAILS;
```

Comment:
This file is not necessary

Reply:
I have used this for pnc outcome details

Reply:
you can also remove the `PncOutcomeDetailsRepository`, but i guess you can include this to to issue 11

Reply:
just ignore it right now because i have worked on that file in issue 11 and if I remove this one then this branch will conflict with issue 11 branch.